src/core/server/logging/layouts/pattern_layout.ts

```diff
@@ -32,14 +32,15 @@ const Parameters = Object.freeze({
   Level: '{level}',
   Message: '{message}',
   Timestamp: '{timestamp}',
+  Pid: '{pid}',
 });
 
 /**
  * Regular expression used to parse log message pattern and fill in placeholders
  * with the actual data.
  */
 const PATTERN_REGEX = new RegExp(
-  `${Parameters.Timestamp}|${Parameters.Level}|${Parameters.Context}|${Parameters.Message}`,
+  `${Parameters.Timestamp}|${Parameters.Level}|${Parameters.Context}|${Parameters.Message}|${Parameters.Pid}`,
   'gi'
 );
```

Comment:
NIT: Using something like `Object.values(Parameters).concat('|')` may be more maintainable.

Reply:
will add in the next PR, where I add another parameter 😅